.gitignore

```diff
@@ -44,3 +44,4 @@ testem.log
 Thumbs.db
 
 TESTS-*.xml
+/angular-reactive-validation/node_modules/*
```

Comment:
**thought**: Probably you ran `npm install` within `/angular-reactive-validation`. There's no need to do so. Run `npm install` in the root and remove the `node_modules` folder in the subfolder. Everything should still work and this line can be removed again 😀.

Reply:
Yes i did. Fixed.